### ISSUE TITLE
Libraries and pkgconfig file need to install to /usr/lib64 on x86_64 systems

### ISSUE THREAD

**Issue by rworkman** [nih-at/libzip#19]
I've patched it here with the following, and while it's not useful for inclusion as-is, it can give an easy starting point:
```
diff -Nur libzip-1.4.0.orig/CMakeLists.txt libzip-1.4.0/CMakeLists.txt
--- libzip-1.4.0.orig/CMakeLists.txt    2017-12-29 06:37:22.000000000 -0600
+++ libzip-1.4.0/CMakeLists.txt 2017-12-29 15:00:54.745740270 -0600
@@ -141,7 +141,7 @@
 ADD_DEFINITIONS("-DHAVE_CONFIG_H")
 
 # rpath handling: use rpath in installed binaries
-SET(CMAKE_INSTALL_RPATH "${CMAKE_INSTALL_PREFIX}/lib")
+SET(CMAKE_INSTALL_RPATH "${CMAKE_INSTALL_PREFIX}/lib@LIBSUFFIX@")
 SET(CMAKE_INSTALL_RPATH_USE_LINK_PATH TRUE)
 
 # Testing
@@ -157,7 +157,7 @@
 # pkgconfig file
 SET(prefix ${CMAKE_INSTALL_PREFIX})
 SET(exec_prefix \${prefix})
-SET(libdir \${exec_prefix}/lib)
+SET(libdir \${exec_prefix}/lib@LIBSUFFIX@)
 SET(includedir \${prefix}/include)
 IF(BZIP2_FOUND)
   SET(LIBS "${LIBS} -lbz2")
@@ -167,7 +167,7 @@
   SET(PKG_CONFIG_RPATH "-Wl,-R\${libdir}")
 ENDIF(CMAKE_SYSTEM_NAME MATCHES BSD)
 CONFIGURE_FILE(libzip.pc.in libzip.pc @ONLY)
-INSTALL(FILES ${CMAKE_CURRENT_BINARY_DIR}/libzip.pc DESTINATION lib/pkgconfig)
+INSTALL(FILES ${CMAKE_CURRENT_BINARY_DIR}/libzip.pc DESTINATION lib@LIBSUFFIX@/pkgconfig)
 
 # write out config file
 CONFIGURE_FILE(${CMAKE_CURRENT_SOURCE_DIR}/cmake-config.h.in ${CMAKE_CURRENT_BINARY_DIR}/config.h)
diff -Nur libzip-1.4.0.orig/lib/CMakeLists.txt libzip-1.4.0/lib/CMakeLists.txt
--- libzip-1.4.0.orig/lib/CMakeLists.txt        2017-12-29 06:37:22.000000000 -0600
+++ libzip-1.4.0/lib/CMakeLists.txt     2017-12-29 15:01:29.343028229 -0600
@@ -204,6 +204,6 @@
 TARGET_LINK_LIBRARIES(zip ${ZLIB_LIBRARY} ${OPTIONAL_LIBRARY})
 INSTALL(TARGETS zip
   RUNTIME DESTINATION bin
-  ARCHIVE DESTINATION lib
-  LIBRARY DESTINATION lib)
+  ARCHIVE DESTINATION lib@LIBSUFFIX@
+  LIBRARY DESTINATION lib@LIBSUFFIX@)
 #CREATE_LIBTOOL_FILE(zip lib)

```

**Comment by 0-wiz-0** [comment]
Fixed in https://github.com/nih-at/libzip/commit/42a3c4a40b9925b8ef0fb8f38b200fdc9f6fe5a4
Define `CMAKE_INSTALL_LIBDIR` to the appropriate value for your environment.

**Comment by kyrios123** [comment]
Doesn't work for me with libzip 1.4.0. 
I added `-DCMAKE_INSTALL_LIBDIR=lib64` but still everything is installed under /usr/lib

**Comment by 0-wiz-0** [comment]
That is easy to explain - the fix is not in 1.4.0, I just committed it.

**Comment by kyrios123** [comment]
Duh stupid me !!
Thanks.
